src/util/eased_variable.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import {easeCubicInOut} from './util.js';
+
+/**
+ * An object for maintaining just enough state to ease a variable.
+ *
+ * @private
+ */
+class EasedVariable {
```

Comment:
a few unit tests for this would be nice!

Reply:
Definitely. I was holding out hope that someone would say "styles implement this exact functionality; you should use this helper: …", but if it seems reasonable, will lock it down 👍